### PR TITLE
improve security by checking if user is logged in with auth2

### PR DESCRIPTION
**Pull request by hjohns28**

### CODE REVIEW

File: frontend/src/Schedulr.js [hunk 1]
```diff
@@ -175,6 +176,20 @@ class Schedulr extends React.Component {
     }));
   }
 
+  componentDidMount() {   
+    window.gapi.load('auth2', () => {
+      this.auth2 = gapi.auth2.init({
+        client_id: '119851197452-crnk45b5i6gsi5povitstfpd203n7j6b.apps.googleusercontent.com',
```
Comment (maceonthompson) [inline frontend/src/Schedulr.js:187]: Is it okay for us to include this or do we need to obfuscate it?

**Review by maceonthompson — APPROVED** [review]
LGTM pending the comment on Schedulr.js

**Review by achillesecos — APPROVED** [review]
LGTM

File: frontend/src/Schedulr.js (hunk 1 above)
Comment (hjohns28) [reply frontend/src/Schedulr.js:187]: I looked it up and I believe that the client id doesn't need to be hidden (the client secret does obviously), so we should be good, but if needed I will change and obfuscate it. Thanks!

**Review by FabriceCastel — APPROVED** [review]
LGTM

**Pull request merged**